mlibrary/pride
# April 2023 dependency updates.

# Overview
Monthly dependency updates to help maintain Pride.

## NPM
`@babel/core` have been updated to their latest versions (as of April 3, 2023).

## Testing
- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`).
- Make sure the PR is consistent in these browsers:
- Install the updated packages.
- Make JavaScript changes.
  - Build the repository.
  - Make sure everything works.
